### PR TITLE
docs: improve configuration file content

### PR DESCRIPTION
**Pull request by ldez**
- Adding a capital letter at the beginning of the sentence and a period at the end of the sentence.
- Splitting lines for better readability.
- Applies a common pattern for the default values.

It's not perfect, but it's another step.

### CODE REVIEW

File: .golangci.example.yml [hunk 1]
```diff
@@ -538,26 +601,105 @@ linters-settings:
         truncate: "32"
 
   govet:
-    # report about shadowed variables
+    # Report about shadowed variables.
     check-shadowing: true
```
Comment (leonklingele) [inline .golangci.example.yml:605]: The default value is `false`.

File: .golangci.example.yml [hunk 2]
```diff
@@ -1134,52 +1308,283 @@ linters-settings:
...
+    - golint
+    - gomnd
+    - gomoddirectives
+    - gomodguard
+    - goprintffuncname
+    - gosec
+    - gosimple
     - govet
+    - ifshort
+    - importas
```
Comment (leonklingele) [inline .golangci.example.yml:1427]: `no-unaliased` of `importas` is `false` by default, yet the config says it's `true`.

File: .golangci.example.yml [hunk 3]
```diff
@@ -1028,13 +1188,13 @@ linters-settings:
     http-status-code-whitelist: [ "200", "400", "404", "500" ]
 
   tagliatelle:
-    # check the struck tag name case
+    # Check the struck tag name case.
     case:
-      # use the struct field name to check the name of the struct tag
+      # Use the struct field name to check the name of the struct tag.
       use-field-name: true
```
Comment (leonklingele) [inline .golangci.example.yml:1194]: `use-field-name` is `false` by default.

File: .golangci.example.yml [hunk 4]
```diff
@@ -196,109 +216,112 @@ linters-settings:
...
-    # See https://go-critic.github.io/overview#checks-overview
-    # To check which checks are enabled run `GL_DEBUG=gocritic golangci-lint run`
-    # By default list of stable checks is used.
+    # Which checks should be enabled; can't be combined with 'disabled-checks'.
+    # See https://go-critic.github.io/overview#checks-overview.
+    # To check which checks are enabled run `GL_DEBUG=gocritic golangci-lint run`.
+    # By default, list of stable checks is used.
     enabled-checks:
       - nestingReduce
       - unnamedresult
```
Comment (leonklingele) [inline .golangci.example.yml:313]: This check is called `unnamedResult` (using an uppercase `R`).

File: .golangci.example.yml [hunk 5]
```diff
@@ -473,36 +519,51 @@ linters-settings:
...
+      # Set to true to raise lint issues for packages that are loaded from a local path via replace directive.
+      local_replace_directives: false
 
   gosimple:
-    # Select the Go version to target. The default is '1.13'.
+    # Select the Go version to target.
+    # Default: 1.13
     go: "1.15"
     # https://staticcheck.io/docs/options#checks
     checks: [ "all" ]
```
Comment (leonklingele) [inline .golangci.example.yml:569]: As far as I can tell, the default is `checks: [ "all", "-ST1000", "-ST1003", "-ST1016", "-ST1020", "-ST1021", "-ST1022", "-ST1023" ]`

File: .golangci.example.yml (hunk 2 above)
Comment (ldez) [reply .golangci.example.yml:1427]: the config is not the default value, the comments contain the default value.

**Review by butuzov — APPROVED** [review]
LGTM

**Pull request merged**